HyBidDemo/HyBidDemoObjc/Controllers/VideoInterstitialViewController.m

```diff
@@ -53,7 +53,7 @@ - (void)requestAd {
     self.videoInterstitialAd.allowAutoPlay = YES;
     self.videoInterstitialAd.allowAudioOnStart = YES;
     self.videoInterstitialAd.delegate = self;
-    VWVideoAdRequest * adRequest = [VWVideoAdRequest requestWithContentCategoryID:VWContentCategoryNewsAndInformation];
+    VWVideoAdRequest * adRequest = [VWVideoAdRequest requestWithContentCategoryIDs:@[@(VWContentCategoryPets), @(VWContentCategoryMusic), @(VWContentCategorySports)]];
     adRequest.minDuration = @(10);
```

Comment:
I think the swift app may be broke by this change. Can you add it there also ?

Reply:
It wont broke since the changes that we made support both single category content IDs and multiple. (In `BannerViewController` for example we still have 1 content category to initialize the `VWAdRequest`.)

`VWAdRequest* adRequest = [VWAdRequest requestWithContentCategoryID:VWContentCategoryNewsAndInformation];`